lib/shared/config-manager/src/index.ts

```diff
@@ -136,7 +150,8 @@ export class EnvironmentConfigManager {
             )
         } else if (responseError?.status === 403) {
             this.stopPolling()
-            throw new UserError(`Invalid SDK key provided: ${this.sdkKey}`)
+            // UserError
```

Comment:
This gets re-added in one of the next stacked PRs.

Reply:
this would affect the behaviour of the SDK, not sure how you can safely make this change even if its put back later

Reply:
From what I can see it doesn't actually affect the behaviour of the SDK, and will be re-added in the next PR where it it is moved to the new request lib.

Reply:
I can manually copy it in here temporarily.